### DISCUSSION TITLE
fix: I18N consistency.

### DISCUSSION THREAD

**Discussion by g4challenge** [open-webui/open-webui#6201] [General]
# Bug Report

**Important: Before submitting a bug report, please check whether a similar issue or feature request has already been posted in the Issues or Discussions section. It's likely we're already tracking it. In case of uncertainty, initiate a discussion post first. This helps us all to efficiently focus on improving the project.**

**Let's collaborate respectfully. If you bring negativity, please understand our capacity to engage may be limited. If you're open to learning and communicating constructively, we're more than happy to assist you. Remember, Open WebUI is a volunteer-driven project maintained by a single maintainer, supported by our amazing contributors who also manage full-time jobs. We respect your time; please respect ours. If you have an issue, We highly encourage you to submit a pull request or to fork the project. We actively work to prevent contributor burnout to preserve the quality and continuity of Open WebUI.**

## Installation Method

Docker

## Environment

- **Open WebUI Version:** 0.3.32

- **Operating System:** macos
- **Browser (if applicable):** Chrome

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [X] I have included the browser console logs.
- [X] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

The new Icons do not internationalize in their respective branding, as well as timezones should have a locale in the browser (24:00 format or the likes):
<img width="244" alt="image" src="https://github.com/user-attachments/assets/1e372523-e23e-470b-8846-e81e36d58802">

Also Artifacts have no text i18n:
<img width="207" alt="image" src="https://github.com/user-attachments/assets/34032671-9875-4149-ad0f-50610d0b1cc7">

as well do temporary chat:
<img width="496" alt="image" src="https://github.com/user-attachments/assets/d60f5b4e-3aa3-40ce-b15a-94f6b7950cec">

and also in the new knowledge search:
<img width="252" alt="image" src="https://github.com/user-attachments/assets/7a1a2ec4-c1ce-43d5-adb6-b68600aef15a">

<img width="584" alt="image" src="https://github.com/user-attachments/assets/b1639d00-039f-4d3e-9a4f-eafeec87f5a6">

## Description

**Bug Summary:**
Missing I18N does not allow native Speakers to fully comprehend the UI they are presented with.

## Reproduction Details

**Steps to Reproduce:**
Simple and straight

## Logs and Screenshots

**Browser Console Logs:**
not applicable

**Docker Container Logs:**
not applicable

**Screenshots/Screen Recordings (if applicable):**
see in text

**Comment by tjbck** [comment]
We rely on our community members for internationalisation, PR welcome to https://github.com/open-webui/open-webui/tree/dev/src/lib/i18n/locales !
